lib/sqlalchemy/orm/relationships.py

```diff
@@ -271,6 +276,19 @@ def _resolve_against_registry(
             self.resolved = attr_value
 
 
+@dataclasses.dataclass
+class _StringRelationshipArg(_RelationshipArg[_T1, _T2]):
+    def _resolve_against_registry(
+        self, clsregistry_resolver: Callable[[str, bool], _class_resolver]
```

Comment:
**mike bayer** (zzzeek) wrote:

test message

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4831

Reply:
**mike bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4831